### ISSUE TITLE
arc

### ISSUE THREAD

**Issue by kig** [play-co/webgl-2d#15]
arc can be done either by generating circle geometry with specified angle delta between two segments or by drawing a quad and using a fragment shader to reject pixels outside the circle (antialias by controlling opacity based on how much of a sample is inside the circle).

**Comment by nickdesaulniers** [comment]
Yes!  If I can find time, I'd love to take a crack at this.  This code was a really good read, and taught me a thing or two.
